sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -498,6 +518,39 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
       else:
         return_type = typehints.Any
       state.stack[-pop_count:] = [return_type]
+    elif opname == 'CALL':
+      pop_count = 1 + arg
+      # Keyword Args case
+      if state.kw_names is not None:
+        if isinstance(state.stack[-pop_count], Const):
+          from apache_beam.pvalue import Row
+          if state.stack[-pop_count].value == Row:
+            fields = state.kw_names
+            return_type = row_type.RowTypeConstraint.from_fields(
+                list(
+                    zip(fields,
+                        Const.unwrap_all(state.stack[-pop_count + 1:]))))
+          else:
+            return_type = Any
+        state.kw_names = None
+      else:
+        # Catch lambdas not passing in arg values properly
```

Comment:
do we already have tests for this?
For my education, what does not passing properly mean in this context?